src/main/java/picard/vcf/VcfToIntervalList.java

```diff
@@ -81,10 +81,17 @@
     @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The output Picard Interval List.")
     public File OUTPUT;
 
-    @Argument(doc="Controls the naming of the resulting intervals. When true, each resulting interval will be named the concatenation of " +
-            "the variant ID fields (if present), or 'interval-<number>' (if not) with a pipe '|' separator. " +
-            "When false, only the first name will be used.")
-    public boolean CONCATENATE_IDS = false;
+
+    public enum VARIANT_ID_TYPES {
+        CONCAT_ALL,
+        USE_FIRST
+    }
+
+    @Argument(doc="Controls the naming of the resulting intervals. When set to CONCAT_ALL (the default), each resulting " +
+            "interval will be named the concatenation of the variant ID fields (if present), or 'interval-<number>' " +
+            "(if not) with a pipe '|' separator. If set to USE_FIRST, only the first name will be used.")
+    public VARIANT_ID_TYPES VARIANT_ID_METHOD = VARIANT_ID_TYPES.CONCAT_ALL;
+    public final boolean concatenate_ids = (VARIANT_ID_METHOD == VARIANT_ID_TYPES.CONCAT_ALL);
```

Comment:
this cannot work...the @Argument value is modified after the class is instantiated...so `concatenate_ids` will always be true. move this line into `doWork()`